components/SalesHistoryTable/index.tsx

```diff
@@ -130,9 +148,9 @@ const SalesHistoryTable = ({ tableData, id, error }: Props): JSX.Element => {
   }, [currentUser]);
 
   const getTableContent = () => {
-    return renderedData.map((sale) => {
+    return renderedData.map((sale, i) => {
       return (
-        <TableRow key={sale.sale_id}>
+        <TableRow key={i}>
```

Comment:
Instead of using the index as the `key`, let's use the sale's asset's ID.

<img width="482" alt="Screen Shot 2021-03-26 at 2 41 14 PM" src="https://user-images.githubusercontent.com/32081352/112695485-6ac13f00-8e41-11eb-9da1-024def0f5bab.png">